pages/rides-overview/index.js

```diff
@@ -0,0 +1,16 @@
+import { rides } from "../../lib/rides";
+import React from "react";
+import { StyledCard } from "../../components/Card/Card.styled";
+import Heading from "../../components/Heading";
+import SecondHeading from "../../components/SecondHeading";
+import Image from "next/image";
```

Comment:
Same as above. You can probably remove this import as no Image needed for this component.